examples/ewdmet/65-callback-solver-dyson.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import pyscf
+import pyscf.gto
+import pyscf.scf
+import pyscf.fci
+import vayesta
+import vayesta.ewf
+from vayesta.misc.molecules import ring
+from dyson import FCI
+
+
+# User defined FCI solver - takes pyscf mf as input and returns RDMs
+# The mf argment contains the hamiltonain in the orthonormal cluster basis
+# Pyscf or other solvers may be used to solve the cluster problem and may return RDMs, CIID amplitudes or CCSD amplitudes
```

Comment:
CIID -> CISD
Also, can you add "or (in this case) the Greens function moments of the cluster", to make it clear what is going on in this example.